src/types/NodeConfig.generated.ts

```diff
@@ -628,7 +642,7 @@ export type NodeConfig = {
       'memory-free-limit'?: string;
       /** pending upgrade wasm module root to additionally validate (hash, 'latest' or empty) (default "latest") */
       'pending-upgrade-module-root'?: string;
-      /** record that many blocks ahead of validation (larger footprint) (default 40) */
+      /** record that many blocks ahead of validation (larger footprint) (default 32) */
```

Comment:
strangely i am getting this when running the generation against this branch
<img width="521" alt="image" src="https://github.com/user-attachments/assets/cd09426a-aa44-4555-9e20-e875524a69b8" />